.github/workflows/release-asset-audit.py

```diff
@@ -0,0 +1,40 @@
+import github
+import sys
+
+token = sys.argv[1]
+
+gh = github.Github(login_or_token=token)
+repo = gh.get_repo('llvm/llvm-project')
+
+uploaders = set([
+    'DimitryAndric',
+    'stefanp-ibm',
+    'lei137',
+    'omjavaid',
+    'nicolerabjohn',
+    'amy-kwan',
+    'mandlebug',
+    'zmodem',
+    'androm3da',
+    'tru',
+    'rovka',
+    'rorth',
+    'quinnlp',
+    'kamaub',
+    'abrisco',
+    'jakeegan',
+    'maryammo',
+    'tstellar',
+    'github-actions[bot]'
+])
+
+for release in repo.get_releases():
```

Comment:
Nit: Probably doesn't really matter, but this could probably be wrapped in a function that is called from a `if __name__ == '__main__'` block.

Reply:
Is this a standard python convention?  I haven't been doing it in any of the other scripts.

Reply:
I don't know about a standard python convention, but it's pretty common and a recommended practice from what I've seen. The Google style guide at least recommends it, citing that tooling like pydoc and testing frameworks need everything to be importable.